airflow/executors/celery_executor.py

```diff
@@ -337,6 +344,12 @@ def sync(self) -> None:
 
         if self.adopted_task_timeouts:
             self._check_for_stalled_adopted_tasks()
+        # Use EventScheduler to avoid checking this too often
+        self.event_scheduler = EventScheduler()
+        self.event_scheduler.call_regular_interval(
+            self.stuck_queued_task_check_interval,
+            self._clear_stuck_queued_tasks(),
+        )
```

Comment:
This isn't quite right -- it will re-create a new "timer" each time `sync()` is called, but never check it -- for that we have the instance live longer than one sync() call and to call `timers.run(blocking=False)`

Reply:
I will create the timer at `start` method. Thanks!